docker/latte/workloads/workload.rn

```diff
@@ -0,0 +1,28 @@
+//! Partition write stress test - writes very tiny single-row partitions.
+
+const INSERT = "insert";
+
+const KEYSPACE = "latte";
+const TABLE = "basic";
+
+pub async fn schema(db) {
+    db.execute(`CREATE KEYSPACE IF NOT EXISTS ${KEYSPACE} \
+                    WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 }`).await?;
+    db.execute(`DROP TABLE IF EXISTS ${KEYSPACE}.${TABLE}`).await?;
```

Comment:
Is dropping required? This is somehow different than c-s I suppose

Reply:
it's not required, it's just copy paste for the tool example.
the idea you can do the flow again and again while clearing the data on every schema creation

in real usage I didn't do that, since it might be called from multiple processes

I'll remove it from here to avoid confusion

Reply:
removed